games-fps/urbanterror/urbanterror-4.3.4_p20211208.ebuild

```diff
@@ -1,34 +1,37 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="7"
+EAPI="8"
+
+COMMIT_ID="c988f7669bd70b59beb384563d4c8304062ae011"
 
 inherit desktop flag-o-matic toolchain-funcs xdg-utils
 
 DESCRIPTION="Hollywood tactical shooter based on the ioquake3 engine"
-HOMEPAGE="https://urbanterror.info https://github.com/mickael9/ioq3"
+HOMEPAGE="https://www.urbanterror.info/ https://github.com/mickael9/ioq3"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/mickael9/ioq3.git"
 	EGIT_BRANCH="urt"
 else
-	COMMIT_ID="0429c03056720523d27ca71d5a4aa3e8d00709e7"
 	SRC_URI="https://github.com/mickael9/ioq3/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
 	S="${WORKDIR}/ioq3-${COMMIT_ID}"
-	KEYWORDS="~amd64 ~x86"
 fi
 
+KEYWORDS="~amd64 ~x86"
 LICENSE="GPL-2"
 SLOT="0"
 IUSE="+altgamma +client +curl debug mumble openal +opus server +skeetshootmod voip vorbis"
-REQUIRED_USE="|| ( client server )
-		voip? ( opus )"
+REQUIRED_USE="
+	|| ( client server )
+	voip? ( opus )
+"
 
 DOCS=( ChangeLog README.md README.ioq3.md md4-readme.txt )
 
 PATCHES=(
-	"${FILESDIR}"/${PN}-4.3.4_p20180708-fix-build_system.patch
+	"${FILESDIR}/${PN}-4.3.4_p20180708-fix-build_system.patch"
```

Comment:
You don't have to change the quote style here, as `PN` is guranteed to contain no space character. Hence, in the spirit of keeping the diff minimal, I suggest to revert that.

Reply:
If you insist on that I'll revert it, however I find that quoting style ending in the middle of a path a bit irritating and prefer to see the quotation marks at first glance. (I know there's syntax highlighting etc, but sometimes you just see raw text.)

Reply:
Yeah, I am also not a fan of that particular style, as I am not aware of any advantages. But I would prefer to keep the existing ebuild where possible. That said, it is not a blocker for me.